### DISCUSSION TITLE
Startup Looping

### DISCUSSION THREAD

**Discussion by parallaxview** [haugene/docker-transmission-openvpn#2397] [Q&A] [answered]
I cannot for the life of me figure out why my startup is looping since the last update, it has been rock solid for a few years up until now. log output:

`Provider NORDVPN has a bundled setup script. Defaulting to internal config

Executing setup script for nordvpn

INFO: OVPN: Checking curl installation

INFO: OVPN: DNS resolution ok

INFO: OVPN: ok, configurations download site reachable

INFO: OVPN: Removing existing configs in /etc/openvpn/nordvpn

INFO: OVPN: server : us9963.nordvpn.com, load: 13

INFO: OVPN: Downloading config: us9963.nordvpn.com.ovpn

INFO: OVPN: Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_tcp/servers/us9963.nordvpn.com.tcp.ovpn

Starting OpenVPN using config us9963.nordvpn.com.ovpn

Modifying /etc/openvpn/nordvpn/us9963.nordvpn.com.ovpn for best behaviour in this container

Modification: Point auth-user-pass option to the username/password file

Modification: Change ca certificate path

Modification: Change ping options

Modification: Update/set resolv-retry to 15 seconds

Modification: Change tls-crypt keyfile path

Modification: Set output verbosity to 3

Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop

Modification: Updating status for config failure detection

Setting OpenVPN credentials...

adding route to local network 192.168.0.0/16 via 172.18.0.1 dev eth0

/etc/openvpn/start.sh: line 321: OPENVPN_OPTS: unbound variable

Starting container with revision: f9cb4dea2da1a3aa63bd3945e0162c9b8a9789a4

Creating TUN device /dev/net/tun

Using OpenVPN provider: NORDVPN

Running with VPN_CONFIG_SOURCE auto

Provider NORDVPN has a bundled setup script. Defaulting to internal config

Executing setup script for nordvpn

INFO: OVPN: Checking curl installation

INFO: OVPN: DNS resolution ok

INFO: OVPN: ok, configurations download site reachable

INFO: OVPN: Removing existing configs in /etc/openvpn/nordvpn

INFO: OVPN: server : us9963.nordvpn.com, load: 13

INFO: OVPN: Downloading config: us9963.nordvpn.com.ovpn

INFO: OVPN: Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_tcp/servers/us9963.nordvpn.com.tcp.ovpn

Starting OpenVPN using config us9963.nordvpn.com.ovpn

Modifying /etc/openvpn/nordvpn/us9963.nordvpn.com.ovpn for best behaviour in this container

Modification: Point auth-user-pass option to the username/password file

Modification: Change ca certificate path

Modification: Change ping options

Modification: Update/set resolv-retry to 15 seconds

Modification: Change tls-crypt keyfile path

Modification: Set output verbosity to 3

Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop

Modification: Updating status for config failure detection

Setting OpenVPN credentials...

adding route to local network 192.168.0.0/16 via 172.18.0.1 dev eth0

/etc/openvpn/start.sh: line 321: OPENVPN_OPTS: unbound variable`

**Comment by Solamain** [comment, marked as answer]
The way I fixed this same issue was placing OPENVPN_OPTS: "" (depending on your compose file you may need to use - OPENVPN_OPTS= without the double quotes instead) in my compose file just after OPENVPN_PASSWORD: 
Hope this helps someone.

**Comment by haugene** [comment]
See https://github.com/haugene/docker-transmission-openvpn/discussions/2395

This should now be fixed if you pull the latest image. I'll make it part of the 4.3 release that I plan to create in a few days. But for now it's on both latest and dev tags.
